routers.go

```diff
@@ -36,6 +38,25 @@ func newObjectLayer(exportPaths ...string) (ObjectLayer, error) {
 
 // configureServer handler returns final handler for the http server.
 func configureServerHandler(srvCmdConfig serverCmdConfig) http.Handler {
+	// FIXME: currently we don't check single exportPath which uses FS layer.
+	if len(srvCmdConfig.exportPaths) > 1 {
+		if isFormatConfigFileExists() {
+			format, err := getFormatXL()
+			if err != nil {
+				fatalIf(err, "Failed to read format.json", nil)
+			}
+
+			if !reflect.DeepEqual(format.Disks, srvCmdConfig.exportPaths) {
+				fatalIf(fmt.Errorf("Disks are not same with previous run"), "", nil)
```

Comment:
"Number of export paths from command-line did not match the backend configuration. Backend is configured with [%s] exports."